src/test/java/uk/gov/hmcts/reform/sscs/service/DwpAddressLookupServiceTest.java

```diff
@@ -266,7 +266,7 @@ public void givenAUcBenefitType_thenDefaultDwpRegionalCenter() {
     public void givenACarersAllowanceBenefitType_thenDefaultDwpRegionalCenter() {
```

Comment:
Not sure if relevant, but do you need a test for the getDwpRegionalCenterByBenefitTypeAndOffice method as well?

Reply:
As there is only one office mapping for carers allowance, it doesn't make any diffence. But I added one in case.